src/images/image-component.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react'
+import {Modal, Button} from 'carbon-components-react'
+import classes from './images-component.scss'
+
+const Images = ({url, fileName}) => {
```

Comment:
Images and Image-component does not represent proper component or feature naming... I believe we would want to support `document-preview` as a feature and within it you would have an `image-preview-component` or `pdf-preview-componet`?

Reply:
We have renamed it to `image-preview-component`